src/chrome/content/rules/CMBChina.com.xml

```diff
@@ -1,39 +1,139 @@
+		<test url="http://office.cmbchina.com/OAMS/Portal/Help/UpdateAlert.htm" />
 	<target host="sdc.cmbchina.com" />
+	<target host="site.cmbchina.com" />
+	<target host="ebank.sz1.cmbchina.com" />
+	<target host="ebank.sz2.cmbchina.com" />
+	<target host="img.trip.cmbchina.com" />
+	<target host="js.trip.cmbchina.com" />
+	<target host="user.cmbchina.com" />
+	<target host="xiaozhao.wx.cmbchina.com" />
+	<target host="xyk.cmbchina.com" />
+	<target host="xykstatic.cmbchina.com" />
+
+	<!--	Complications:	-->
 	<target host="www.cmbchina.com" />
-		<exclusion pattern="^http://www\.cmbchina\.com/$" />
-		<test url="http://www.cmbchina.com/" />
+	<exclusion pattern="^http://www\.cmbchina\.com/$" />
+		<test url="http://www.cmbchina.com/personal/" />
 
-	<rule from="^http:" to="https:" />
+	<target host="ccclub.cmbchina.com" />
+	<exclusion pattern="^http://ccclub\.cmbchina\.com/$" />
+		<test url="http://ccclub.cmbchina.com/ccproduct/fushu.aspx" />
+
+	<target host="forum.cmbchina.com" />
+	<exclusion pattern="^http://forum\.cmbchina\.com/\S+\.aspx" />
+		<test url="http://forum.cmbchina.com/CmbMobileCredictApply/Default.aspx" />
+		<test url="http://forum.cmbchina.com/cmu/viewforum.aspx?forumid=50" />
+
+		<test url="http://forum.cmbchina.com/cmbwebshare/Scripts/CmbWebShare.js" />
+		<test url="http://forum.cmbchina.com/CMU/images/tp0000.gif" />
+		<test url="http://forum.cmbchina.com/cmu/images/announce.gif" />
+		<test url="http://forum.cmbchina.com/cmu/scripts/default.js" />
+		<test url="http://forum.cmbchina.com/cmu/style/default.css" />
+		<test url="http://forum.cmbchina.com/webpages/firmbankweb/zhiyin.htm" />
+		<test url="http://forum.cmbchina.com/webpages/fxexchange/index.html" />
+
+	<target host="img.jf.cmbchina.com" />
+	<target host="ssl.img.jf.cmbchina.com" />
+	<rule from="^http://(ssl\.)?img\.jf\.cmbchina\.com/"
+			to="https://img.jf.cmbchina.com/" />
+		<test url="http://img.jf.cmbchina.com/images/CM/P60/A0C-706-021.jpg" />
+		<test url="http://ssl.img.jf.cmbchina.com/images/CM/P60/A0C-706-021.jpg" />
 
+	<target host="live.cmbchina.com" />
+	<exclusion pattern="^http://live\.cmbchina\.com/\S+\.aspx" />
+	<rule from="^http://live\.cmbchina\.com/"
+			to="https://forum.cmbchina.com/" />
+		<test url="http://live.cmbchina.com/CmbMobileCredictApply/Default.aspx" />
+		<test url="http://live.cmbchina.com/cmu/viewforum.aspx?forumid=50" />
+
+		<test url="http://live.cmbchina.com/cmbwebshare/Scripts/CmbWebShare.js" />
+		<test url="http://live.cmbchina.com/CMU/images/tp0000.gif" />
+		<test url="http://live.cmbchina.com/cmu/images/announce.gif" />
+		<test url="http://live.cmbchina.com/cmu/scripts/default.js" />
+		<test url="http://live.cmbchina.com/cmu/style/default.css" />
+		<test url="http://live.cmbchina.com/webpages/firmbankweb/zhiyin.htm" />
+		<test url="http://live.cmbchina.com/webpages/fxexchange/index.html" />
+
+	<target host="market.cmbchina.com" />
+	<exclusion pattern="^http://market\.cmbchina\.com/ccard/business/" />
```

Comment:
You exclude only `/ccard/business/`, but it seems that there are other affected pages.